python/ray/serve/_private/proxy_request_response.py

```diff
@@ -139,7 +139,7 @@ def request_type(self) -> str:
 
     @property
     def method(self) -> str:
-        return "GRPC"
+        return self.method_name
```

Comment:
@GeneDer can we actually include both _service_ and method name like following `service/method`